include/zenoh-pico/collections/refcount.h

```diff
@@ -244,6 +244,9 @@ size_t _z_simple_rc_strong_count(void *cnt);
         *p = name##_simple_rc_null();                                                                         \
         return res;                                                                                           \
     }                                                                                                         \
+    static inline bool name##_simple_rc_is_last_ref(const name##_simple_rc_t *p) {                            \
```

Comment:
I think it would be more useful to have a function that returns the reference count.